maliput/tests/branch_point_tests.rs

```diff
@@ -34,17 +34,17 @@ fn branch_point_api() {
     let road_network = common::create_t_shape_road_network();
     let road_geometry = road_network.road_geometry();
     let branch_point_id = String::from("2");
-    let branch_point = road_geometry.get_branch_point(&branch_point_id);
+    let branch_point = road_geometry.get_branch_point(&branch_point_id).unwrap();
     assert_eq!(branch_point.id(), branch_point_id);
     assert_eq!(branch_point.road_geometry().id(), road_geometry.id());
     // Testing that the api works. The actual values are not important, they are tested in the
     // cpp tests.
     let lane_end_set = branch_point.get_a_side();
     assert_eq!(lane_end_set.size(), 1);
-    let lane_end = lane_end_set.get(0);
-    let confluent_branches = branch_point.get_confluent_branches(&lane_end);
+    let lane_end = lane_end_set.get(0).unwrap();
```

Comment:
Can we add a test for when you use an index that doesn't return a valid lane_end?

Reply:
Done.